internal/pkg/template/templates/addons/ddb/env/access_policy.yml

```
@@ -0,0 +1,54 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: Your workload's name.
+Resources:
+  {{logicalIDSafe .Name}}AccessPolicy:
```

Should this be something like:
```suggestion
  {{logicalIDSafe .Name}}DdbTableAccessPolicy:
```